pkg/types/types.go

```diff
@@ -83,6 +83,7 @@ type VulnerabilityDetail struct {
 	Description      string     `json:",omitempty"`
 	PublishedDate    *time.Time `json:",omitempty"` // Take from NVD or GHSA
 	LastModifiedDate *time.Time `json:",omitempty"` // Take from NVD or GHSA
+	VulnStatus       string     `json:",omitempty"`
```

Comment:
I would change it to `Status`.

```suggestion
	Status       string     `json:",omitempty"` // Rejected or not
```